repose-aggregator/components/filters/keystone-v2/src/main/scala/org/openrepose/filters/keystonev2/KeystoneV2Filter.scala

```diff
@@ -626,6 +626,7 @@ class KeystoneV2Filter @Inject()(configurationService: ConfigurationService,
       // Removes an extra slash at the end of the URI if applicable
       val serviceUri = keystoneV2Config.getIdentityService.getUri
       keystoneV2Config.getIdentityService.setUri(serviceUri.stripSuffix("/"))
+      akkaServiceClient.setConnectionPoolId(keystoneV2Config.getIdentityService.getConnectionPoolId)
```

Comment:
this isn't safe. the client is used by multiple filters and you just changed them all to using this one's configured pool